third-party/qthread/Makefile

```diff
@@ -46,14 +46,19 @@ ifeq (, $(call isTrue, $(CHPL_QTHREAD_NO_CHPL_ALLOC)))
 endif
 
 ifeq ($(CFLAGS_NEEDS_RT_INCLUDES), y)
-  CFLAGS += -I$(CHPL_MAKE_HOME)/runtime/include/localeModels/$(CHPL_MAKE_LOCALE_MODEL)
-  CFLAGS += -I$(CHPL_MAKE_HOME)/runtime/include/mem/$(CHPL_MAKE_MEM)
-  CFLAGS += -I$(CHPL_MAKE_HOME)/runtime/include
-  ifeq (, $(call isTrue, $(CHPL_QTHREAD_NO_CHPL_ALLOC)))
-    ifeq ($(CHPL_MAKE_MEM),jemalloc)
-      CFLAGS += -I$(JEMALLOC_INCLUDE_DIR)
-    endif
-  endif
+  # When building the Chapel allocator file we need all the Chapel runtime
+  # defs and include directories, plus the third-party include dirs other
+  # than that for Qthreads itself (since we're building it).  Notably, we
+  # do not want -DOPTIMIZE; if we have that set when building Qthreads it
```

Comment:
Think it's DNDEBUG, not DOPTIMIZE

Reply:
https://github.com/chapel-lang/chapel/pull/9736/commits/4bffdb5bc0e814d52bde3d93a72acbda18c706a7 :)

Reply:
Crud, I meant to fix that at the time!